src/sql/base/browser/ui/table/tableCellEditorFactory.ts

```diff
@@ -174,4 +175,76 @@ export class TableCellEditorFactory {
 		}
 		return TextEditor;
 	}
+
+	public getDropDownEditorClass(context: any, defaultOptions: string[]): any {
```

Comment:
@abist  is it possible to merge this with getSelectBoxEditorClass method? the only difference from the consumers perspective is a boolean property: editable.